test/test_mahalanobis_mixin.py

```diff
@@ -247,21 +236,8 @@ def test_get_metric_is_pseudo_metric(estimator, build_dataset):
             np.isclose(metric(a, c), metric(a, b) + metric(b, c), rtol=1e-20))
 
 
-@pytest.mark.parametrize('estimator, build_dataset', metric_learners,
-                         ids=ids_metric_learners)
-def test_get_metric_compatible_with_scikit_learn(estimator, build_dataset):
```

Comment:
where did this test go?
it should probably be kept here only because the output of `get_metric` only has the correct semantic (of a distance function) for Mahalanobis learners, not for bilinear learners (which returns a similarity function)

Reply:
I moved it to `test_sklearn_compat.py` as it's sklearn-specific, but yes, it's for Mahalanobis learners only, so I can keep it here

Reply:
I read the comment below